pom.xml

```diff
@@ -2078,7 +2078,10 @@
                 <param>com.puppycrawl.tools.checkstyle.checks.sizes.*</param>
                 <param>com.puppycrawl.tools.checkstyle.checks.whitespace.*</param>
               </targetTests>
-              <mutationThreshold>89</mutationThreshold>
+              <excludedMethods>
+                <param>destroy</param>
```

Comment:
@Nimfadora , reason is required for all suppressions, even temporal, especially for permanent suppressions.